plugin/trino-geospatial/src/main/java/io/trino/plugin/geospatial/GeoFunctions.java

```diff
@@ -1426,7 +1426,16 @@ public static Slice fromGeoJsonGeometry(@SqlType(VARCHAR) Slice input)
     @ScalarFunction("to_geojson_geometry")
     @Description("Returns GeoJSON string based on the input spherical geography")
     @SqlType(VARCHAR)
-    public static Slice toGeoJsonGeometry(@SqlType(StandardTypes.SPHERICAL_GEOGRAPHY) Slice input)
+    public static Slice toGeoJsonGeometryFromGeography(@SqlType(StandardTypes.SPHERICAL_GEOGRAPHY) Slice input)
+    {
+        return Slices.utf8Slice(jsonFromJtsGeometry(JtsGeometrySerde.deserialize(input)));
+    }
+
+    @SqlNullable
+    @ScalarFunction("to_geojson_geometry")
+    @Description("Returns GeoJSON string based on the input geometry")
+    @SqlType(VARCHAR)
+    public static Slice toGeoJsonGeometryFromGeometry(@SqlType(StandardTypes.GEOMETRY) Slice input)
```

Comment:
Similarly, rename this one to `geometryToGeoJson`

Reply:
Done